find-darkbox.lic

```diff
@@ -42,7 +42,17 @@ loop do
   #16315 bank
   fput('play dark')
   pause 1
-  break if Flags['darkbox-no-money']
+  #break if Flags['darkbox-no-money']
+  if Flags['darkbox-no-money']
+    DRCT.walk_to(16_315)
+    case DRC.bput('withdraw 3 platinum', 'we are not lending money at this time', 'The clerk counts out')
+    when 'we are not lending money at this time'
+      exit
```

Comment:
💡 It would be nice to `echo` or `message` to the user that they need to get more money

Reply:
I will add in on the next enhancement. The Darkbox completely disappeared from the map last night. The script ran through trying to find it and then returned to the last known location and tried to play. I am going to add exit if Darkbox is not in the room after it runs everywhere.